trillian/ctfe/handlers.go

```diff
@@ -460,6 +465,19 @@ func addChainInternal(ctx context.Context, li *logInfo, w http.ResponseWriter, r
 	// epoch, and use this throughout.
 	timeMillis := uint64(li.TimeSource.Now().UnixNano() / millisPerNano)
 
+	// If CTFE storage is enabled for issuance chain, add the chain to storage and cache, get the chain hash for Trillian gRPC.
+	if li.issuanceChainService.IsCTFEStorageEnabled() {
+		// TODO: Check how to convert []ct.ASN1Cert to []byte correctly.
+		issuanceChain, err := asn1.Marshal(extractRawCerts(chain)[1:])
```

Comment:
@mhutchinson Does the `gob` package provide the modern marshal/unmarshal methods?

Reply:
It does, but it probably makes sense to stay in the world of asn1 given that we use this everywhere else for CT.

Reply:
Gotcha, removed the TODO. Thanks.